packs/apache/config.yml

```diff
@@ -5,8 +5,35 @@ name: Apache
 
 # Description of the pack
 description: |+
-  Official New Relic pack for Apache.
-  Use this pack together with the New Relic Apache On Host Integration to get insight into the performance of your Apache instances.
+  # Complete Apache Monitoring Kit
```

Comment:
this feels a bit wordy to me from a developer POV, so I'd like to ask to try to reduce the amount of text and provide a more succinct descrption, cc @bradleycamacho for his opinion